python/lsst/ts/salobj/domain.py

```diff
@@ -215,6 +222,40 @@ def add_salinfo(self, salinfo):
             raise RuntimeError(f"salinfo {salinfo} already added")
         self._salinfo_set.add(salinfo)
 
+    def make_publisher(self, partition_names):
+        """Make a dds publisher.
+
+        Parameters
+        ----------
+        partition_names : `list` [`str`]
+            List of DDS partition names.
+        """
+        partition_qos_policy = dds.PartitionQosPolicy(partition_names)
+
+        # Any of the qos sets will dom because the publisher and subscriber
```

Comment:
Not sure I understood this comment. Is it a typo in "will dom"?

Reply:
It should be "do", not "dom". I'll fix it.

Reply:
Fixed